hrm_oo.py

```diff
@@ -0,0 +1,90 @@
+from readHR import EcgReader
+from hrmcalcs2oo import hrmcalcs
+from hrmtb import TachyBrady
+
+
+class HrmOutput:
+
+    def __init__(self, file='Test_ECG.csv', start_min=3, end_min=5,
+                 brady_limit=60, tachy_limit=100):
+        """
+
+        :param file: The file to make heart rate calculations for
```

Comment:
super-nit: Not a huge deal, but may want to make it clear that this is a filename _string_ as opposed to a file object for the folks reading your docs!